java/core/src/main/java/sleeper/core/status/ProcessFinishedStatus.java

```diff
@@ -13,32 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package sleeper.compaction.job.status;
+package sleeper.core.status;
```

Comment:
Should this package be sleeper.core.record.process.status? It seems odd that it's so far separated from the records processed stuff that's really part of it.

Reply:
That works better, was trying to think of names/ways to link them.